README.md

```diff
@@ -1,50 +1,71 @@
+MythXvsc is an extension for running MythX smart contract analysis from Visual Studio Code.
 
 The extensions provides:
 
--   Smart contract compilation (via `solidity` VSCode extension)
--   Log in to MythX platform
+-   Smart contract compilation (via the Solidity VS Code extension)
+-   Authentication to the MythX platform
 -   AST extraction from compiled source
 -   Submission of analysis
+-   Displaying analysis result in VS Code in a linting fashion
 
 ## Instructions
 
-After installing the extension please enter your registered MythX `ethAddress` and `password` in VSCode user settings. These fields are properties of `mythxvsc` as shown in the screenshots below:
+# Access token login (preferred)
 
-![installation](images/screenshots/installation.png)
-![user_settings](images/screenshots/user_settings.png)
+After registering your mythx account, visit [dashboard.mythx.io/](https://dashboard.mythx.io/) and from the tools page create a new access token (also reffered as API key). Please copy the token and paste it in the Mythx extension user settings as shown below.
 
-**Please note that the credentials stored this way are exposed to VSCode. Be sure to understand the security risk this entails or contact the extension developers if you don't. We are working on a more robust log-in implementation.**
+![access_token](images/screenshots/access_token.png)
 
-If no `ethAddress` and `password` provided the extension will fallback to use default trial credentials for MythX.
+# Ethereum Address / password login (not reccomended)
 
-Now simply open a `.sol` file and click on the MythX icon that you will see in the top right of your IDE window. Otherwise right click with your mouse on the contract name and you will see the Analyze command.
-A third option is to run the command pallete (CMD+SHIFT+P on Mac or equivalent) and look for the `MythX: Analyze smart contract` command.
+If you do want to use a ethereum address/password combination the extension supports it. Please enter your registered MythX Ethereum address and password in the VS Code user settings. These fields are properties of the extension as shown in the screenshots below:
 
-![button_one](images/screenshots/button_mythx.png)
-![right_click](images/screenshots/right_click.png)
+After installing the extension please enter your registered MythX `ethAddress` and `password` in VSCode user settings. These fields are properties of `mythxvsc` as shown in the screenshots below.
 
-Once solidity compilation is done you will be prompted to enter the name of the contract you want to analyze from a list of contracts found in the solidity file. It is very important that you pick the main contract name or the API will return an error and your analysis will fail.
+If no `ethAddress` and `password` provided the extension will fallback to use default trial credentials for MythX (trials account will soon be deprecated).
```

Comment:
Should remove this line as we want people to authenticate using the API key and the Trial mode is deprecated.